packages/manager/modules/anthos/src/dashboard/constants.js

```diff
@@ -1,5 +1,7 @@
 export const TRACKING_PREFIX = 'hpc::anthos::project';
+export const SERVICE_TYPE = 'DEDICATED_ANTHOS';
```

Comment:
question : are you sure the requested `serviceType` should not be `dedicated-anthos` ?

Reply:
It is not possible to put the lower case version directly.
I think it's linked to kebabCase.
```
dedicated::account::billing::autorenew::${kebabCase(
  $state.transition.params().serviceType,
)}::delete
```